src/Distortion.cpp

```diff
@@ -265,14 +271,20 @@ void removeDistortion(double dx, double dy, double &ux, double &uy,
     case LROLROCNAC: {
 
       if (opticalDistCoeffs.size() != 1) {
-        throw "Distortion coefficients for LRO LROC NAC must be of size 1, current size: " +  std::to_string(opticalDistCoeffs.size());
+        csm::Error::ErrorType errorType = csm::Error::INDEX_OUT_OF_RANGE;
```

Comment:
I'm not 100% sure this is the correct error type, but it seemed the most appropriate.

Reply:
`SENSOR_MODEL_NOT_CONSTRUCTIBLE` might be a reasonable alternative?